anime/Services/ImportScheduleService.php

```diff
@@ -232,16 +232,22 @@ private function loadMapping() : array {
 
         for ($i = 1 /* skip the header */; $i < count($mappingLines); ++$i) {
             $mappingLine = str_getcsv(trim($mappingLines[$i]));
-            if (count($mappingLine) < 6)
+            if ((count($mappingLine) < 6) && !((count($mappingLine) == 2) && ($mappingLine[1] != '-')))
```

Comment:
To give you an idea of last year's schedule:

https://docs.google.com/a/lvp-media.com/spreadsheets/d/1dEyuNHUCrRqe2Sw0yu0BhIerw4HKHrMsbdD7E_2UwdA/edit?usp=sharing

This year's mappings are pretty much identical. There's a fair bunch of rows that don't have the description, location or floor information set (anything with a set event ID really), but for maintainability of the spreadsheet itself we should maintain at least the first three columns. At that point, is this branch really worth it?